### ISSUE TITLE
RCS1057 False Positive

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#479]
RCS1057 requires a space between A and [System.Obsolete("")] but also a space between B and C. Is the space between B and C a false positive?

```csharp
namespace RCS1057_False_Positive
{
    static class Program
    {
        public static string A { get; set; }
        [System.Obsolete("")]
        public static string B { get; set; }
        public static string C { get; set; }
        public static string D { get; set; }
    }
}
```
